src/main/kotlin/com/frc2036/comp/game/game.kt

```diff
@@ -17,7 +17,7 @@ enum class Direction(val dx: Int, val dy: Int) {
     NorthEast(-1, -1),
     NorthWest(1, -1),
     SouthEast(-1, 1),
-    SouthWest(1, -1);
+    SouthWest(1, 1);
```

Comment:
Southwest is right (+1) and down(+1). It was previously incorrectly the same as NorthWest.